.github/workflows/release_scripts/prisma_vulnurability_checker.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import json
+import os
+import boto3
+import textwrap
+
+PRISMA_ROOT_API_URL = os.getenv("PRISMA_ROOT_API_URL")
+PRISMA_AUTHENTICATE_URL = f"{PRISMA_ROOT_API_URL}/authenticate"
+PRISMA_REGISTRY_SCAN_RESULT_URL = f"{PRISMA_ROOT_API_URL}/registry"
+PRISMA_ACCESS_KEY = os.getenv('PRISMA_ACCESS_KEY')
+PRISMA_ACCESS_KEY_SECRET = os.getenv('PRISMA_ACCESS_KEY_SECRET')
+HTTP_JSON_HEADER = 'application/json'
+# DOCKER_IMAGE_TO_CHECK is the repository name full and the tag that is used to scan it
+DOCKER_IMAGE_TO_CHECK = os.getenv("DOCKER_IMAGE_TO_CHECK")
+PRISMA_BLOCKING_VULNERABILITIES = "critical,high"
+
+
+def get_excluded_packages():
+    dynamodb_client = boto3.resource('dynamodb')
+
+    whitesource_exclusion_table = dynamodb_client.Table('prisma-excluded-packages')
+    whitesource_exclusion_entries = whitesource_exclusion_table.scan()['Items']
+
+    exclusions = set()
+    for exclusion in whitesource_exclusion_entries:
+        exclusions.add(exclusion['package'])
+
+    return exclusions
+
+
+def block_print(long_string, each_line_length=75):
+    print("\n\t".join(textwrap.wrap(long_string, each_line_length)))
+
+
+def find_all_high_critical_vulnerabilities_to_resolve(excluded_libraries):
+    # Authenticate
+    authenticate_payload = {
+        'username': PRISMA_ACCESS_KEY,
+        'password': PRISMA_ACCESS_KEY_SECRET
+    }
+    authenticate_headers = {'Accept': HTTP_JSON_HEADER, 'Content-type': HTTP_JSON_HEADER}
+
+    get_token_request = requests.post(
```

Comment:
Since it will only make one call to prisma cloud to get the results, im good with generating a session token per call in this scenario

Reply:
exactly, for a signle call it's ok, if we want to query several repos, we should generate the token once and re-use the previously generate token in following calls.